lua/architext/edit.lua

```diff
@@ -96,6 +96,6 @@ function M.edit(buf, parser, query, capture_changes, start_row, end_row)
     end
   end
 
-  vim.lsp.util.apply_text_edits(edits, buf)
+  vim.lsp.util.apply_text_edits(edits, buf, vim.api.nvim_buf_get_option(buf,'fileencoding'))
```

Comment:
I think that tree-sitter integration in neovim used UTF8 only for now, so I guess you should set that to UTF8 rather.